app/models/template.rb

```diff
@@ -87,6 +87,7 @@ def import_without_save(text, options = {})
     self.snippet = !!@importing_metadata[:snippet]
     self.default = options[:default] unless options[:default].nil?
     self.description = @importing_metadata[:description]
+    self.locked = options[:locked] unless options[:locked].nil?
     handle_lock_on_import(options)
```

Comment:
note that this line already contains the logic, the problem is the argument name